ingest/anndata_.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, file_path, study_file_id, study_id, **kwargs):
         )
         # If performing cluster extraction, set obsm_keys
         extract_cluster = kwargs.get("extract_cluster")
-        if extract_cluster:
+        if extract_cluster == True:
```

Comment:
If future us decides to manipulate `obsm_keys` at  `AnnDataIngestor` instantiation, we'll want to check for `True` here too.